### PR TITLE
Adds file drop component functionality to starting points menu

### LINKED ISSUE #228: Add Import Profile Button to Starting Points
Add a new React component for an Import Profile Button for dynamically loading a local Profile JSON file made up resourceTemplates.

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #228 
- Load resource template data from file and iterate through all RTs for render.
-- Adds a fixture file `item.json` for loading demo purposes.
- Adds new spoofed resource templates to go with the `item.json` profile.
- Use spread props for typeahead components
- Hide loading spinner by default for lookup/resource components
- allow `info` and `log` console messages

### CODE REVIEW

File: src/sinopiaServerSpoof.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ const transcribedTitleRt = require('../static/spoofedFilesFromServer/fromSinopia
 const varTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/VarTitle.json')
 const workTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkTitle.json')
 const workVariantTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkVariantTitle.json')
+const lccnRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/LCCN.json')
```
Comment (jermnelson) [inline src/sinopiaServerSpoof.js]: I understand we need these hardcoded sources for the Sinopia Server Spoof (and likely will need to continue to support those sources that are not available in QA)

File: src/components/editor/StartingPoints.jsx [hunk 2]
```diff
@@ -34,14 +45,14 @@ class StartingPoints extends Component {
     let startingPoints = {
       border: '1px dotted',
       float: 'left',
-      padding: '20px'
+      padding: '20px',
```
Comment (ndushay) [inline src/components/editor/StartingPoints.jsx]: d'oh!

File: src/sinopiaServerSpoof.js (hunk 1 above)
Comment (ndushay) [reply src/sinopiaServerSpoof.js]: I think we'll be able to replace these spoofed hardcoded ones once we can talk to sinopiaServer ....

**Comment by ndushay** [comment]
i'm gonna merge this, but i sure wish there were pix of the changes in the UI in the PR description ...

**Pull request merged**